### ISSUE TITLE
Add a License

### ISSUE THREAD

**Issue by stefanogermano** [dave90/ANDLV#14]
We should add a _License_ to our project.

I suggest to use the _MIT License_.

Information about how to add a license in GitHub can be found [here](https://help.github.com/articles/open-source-licensing).
Information about OSS licenses can be found [here](http://choosealicense.com).

**Comment by brainatwork** [comment]
I agree with mit license too. I think it's good for the academic context.
